networking/src/ApiClient.ts

```diff
@@ -885,10 +882,11 @@ class QuestionTemplateParser {
             version,
             choices,
             instructions,
-            domain,
-            cluster,
-            grade,
-            standard,
+            answerSettings,
```

Comment:
Since it's a JSON, should we cast it to a strongly type? Might be in another PR though